scheduler/src/cook/mesos/rebalancer.clj

```diff
@@ -485,7 +486,7 @@
             (catch Throwable e
               (log/warn e "Failed to transact preemption")))
           (when-let [task-id (:instance/task-id task-ent)]
-            (mesos/kill-task! driver {:value task-id})))))))
```

Comment:
for instance, this is probably going to break because it's getting an atom now.

Reply:
Yup. Long fixed.